Platform/ElkhartlakeBoardPkg/Library/Stage1ABoardInitLib/Stage1ABoardInitLib.c

```diff
@@ -121,7 +123,7 @@ BoardInit (
     DisableWatchDogTimer ();
     EarlyPlatformDataCheck ();
     if (GetDebugPort () < PCH_MAX_SERIALIO_UART_CONTROLLERS) {
-      GpioConfigurePads (2, (GPIO_INIT_CONFIG *) (&mUartGpioTable[2 * (GetDebugPort () - 1)]));
+      GpioPadConfigTable(2,(VOID *) (&mUartGpioTable[2 * (GetDebugPort () - 1)]));

```

Comment:
Please use ConfigureGpio () here as well. Is there a limitation?

Reply:
Submitted patch with ConfigureGpio replaced